Shadertoy/ShadertoyRender.h

```diff
@@ -1907,7 +1925,8 @@ ShadertoyPlugin::RENDERFUNC(const OFX::RenderArguments &args)
 
     glPushAttrib(GL_ALL_ATTRIB_BITS);
     glDisable(GL_BLEND);
-    glEnable(GL_DEPTH_TEST);
+    glDisable(GL_DEPTH_TEST);
+    glEnable(GL_SCISSOR_TEST);
```

Comment:
why enable GL_SCISSOR? It's really only used for Mesa, which renders by tile (because it's more efficient in the Mesa architecture)

Reply:
I suppose it isn't necessary after all, but the reason I enabled it in the first place is because there's a call to glScissor on line 1973, which didn't make sense to me without the scissor test.

Also, is there a reason why depth testing was enabled?

Reply:
yes, because the original OpenGL code (TestOpenGL.cpp) draws a teapot, which is extra nice. It makes no sense in Shadertoy of course

Reply:
Hah, gotcha :P
I'll revert the scissor test change